support-frontend/assets/helpers/redux/checkout/product/contributionsSideEffects.ts

```diff
@@ -66,20 +66,6 @@ export function addProductSideEffects(
 		},
 	});
 
-	startListening({
-		actionCreator: setProductType,
-		effect(action) {
-			/**
-			 * selectedContributionType is read from storage in 2 places:
-			 * 1) It is used to set the selected contribution type on return
-			 * visits to the Contributions Landing Page in the same session.
-			 * 2) It is used on the Contributions Thank You page to send data to Quantum
```

Comment:
How (or are we even) sending the data to QM now? And what about the cookie

Reply:
I _think_ this might be for if you refresh the page. Reason being we don't store this information in the URL of the thank you page.